SazhnevDjango/SazhnevProject/sazhnevapp/forms.py

```diff
@@ -0,0 +1,8 @@
+from django import forms
+from .models import Product
+
+
+class ProductForm(forms.ModelForm):
+    class Meta:
+        model = Product
+        fields = ['name', 'description', 'price', 'quantity', 'image']
```

Comment:
Если полей, которые вы хотите исключить меньше, чем полей, которые вы хотите добавить в форму, то лучше использовать список exclude.
```suggestion
        exclude = ['added_date']
```

Reply:
Кроме того, можно еще добавить словари `lables` и `widgets`